package.json

```diff
@@ -81,8 +81,9 @@
     "docs-compile": "npm run docs-build",
     "docs-vnu": "node build/vnu-jar.js",
     "docs-lint": "npm run docs-vnu",
-    "docs-serve": "hugo server --port 9001 --disableFastRender",
-    "docs-serve-only": "npx sirv-cli _site --port 9001",
+    "docs-serve": "hugo server --port 9001 --bind=0.0.0.0 --disableFastRender",
+    "docs-serve-only": "npx http-server _site --port 9001",
+    "visual-tests-serve": "npx http-server . --port 9001 -o js/tests/visual",
```

Comment:
http-server has nice features out of the box compared to sirv-cli:

- `-o` option: https://github.com/http-party/http-server#available-options

> Open browser window after starting the server. Optionally provide a URL path to open. e.g.: -o /other/dir/

- it lists the files in directories (no need to write an index.html)

<img width="427" alt="image" src="https://user-images.githubusercontent.com/643434/162641402-ebdc1547-a417-41b3-ab9d-9dd9762bbe85.png">

Reply:
We try to keep our deps to a minimum. And that's the reason we use sirv-cli and we plan to do so.

Reply:
We don't plan to change to http-serve, sorry. Keep your PRs clean, one change at a time. Similarly, we don't want to change to `0.0.0.0`.

Reply:
I've reverted the 2 commits.

Still interested to know what you plan to do with the visual tests.
If polished they can be a great asset and catch lot of problems earlier.

I'm using "visual tests" for 2 libraries based on Bootstrap:
- https://codesandbox.io/s/github/tkrotoff/bootstrap-floating-label/tree/codesandbox.io (https://github.com/tkrotoff/bootstrap-floating-label)
- https://codesandbox.io/s/github/tkrotoff/bootstrap-input-spinner/tree/codesandbox.io (https://github.com/tkrotoff/bootstrap-input-spinner)

They quickly became essential for me for non-regressions and edge cases.

<img width="304" alt="image" src="https://user-images.githubusercontent.com/643434/163041151-0fb0b364-561e-4bf6-aabd-9bb4382a8783.png">